File: 01/RectangleArea.java
/*Input: The height and width
 * Precondition: The height and width is a positive number.
 *  * Output: The area of the rectangle.
 *   *
 *    * Begun by: Dr. Adams, for CS 214, at Calvin College.
 *     * Completed by: Justin Voss
 *      * Date: 1/14/22
 *       **********************************************************/


 import java.util.Scanner;  // include class for Scanner

 public class RectangleArea {

	      /* function circleArea() computes a circle's area, given its radius.
	       *       * Parameter: r, a double
	       *             * Precondition: r is not negative.
	       *                   * Returns: the area of the circle
	       *                         */
	      public static double rectangleArea(double h, double w) {
		              return h * w;               // return an expression
			           } // RectangleArea method
	      	
	     //    main program
		   public static void main(String[] args) {
		         // prompt for height and width
		               System.out.println("\n\nTo compute the area of a rectangle,");
		                     System.out.println(" enter its height and width. ");
                             System.out.print(" Height: ");
		
		                           // Create a connection named keyboard to standard in
		                                 Scanner keyboard = new Scanner(System.in);
		                                       //Get the number from the user
		                                             double height = keyboard.nextDouble();
                                                     System.out.print("\n Width:");
                                                     double width = keyboard.nextDouble();
		                                                   // output area
		                                                         System.out.println("\nThe area is " + rectangleArea(height,width) + "\n");
		                                                               System.out.printf("The area is %f\n\n", rectangleArea(height,width));
		                                                                     System.out.printf("The area is %.15f\n\n", rectangleArea(height,width));
		                                                                       } // main method
		
		                                                                       } // class RectangleArea

File: 01/rectangle_area.rb
#! /usr/bin/ruby
# circle_area.rb computes the area of a rectangle given its height and width
# 
# Input: the height and width of a rectangle
# Precondition: the height and width is not negative
# Output: the area of the rectangle.
#
# Begun by: Prof. Adams, for CS 214 at Calvin College.
# Completed by: Justin Voss
# Date: 1/14/22
###############################################################

# function rectangleArea returns a rectangle's area, given its height and width
# Parameters: height and width, 2 numbers
# Precondition: height, width > 0.
# Returns: the area of a rectangle whose height and width is h and w.
PI = 3.1415927
def rectangleArea(h,w)
    h * w
end

if __FILE__ == $0
   puts "To compute the area of a rectangle,"
   puts " enter its height and width: "
   print " Height: "
   height = gets.chomp.to_f
   print " Width: "
   width = gets.chomp.to_f
   print "The rectangle's area is: "
   puts rectangleArea(height,width)
end